test/functional/test_framework/mempool_util.py

```diff
@@ -19,6 +19,8 @@
     MiniWallet,
 )
 
+ORPHAN_MAX_RETENTION_TIME = 1200
```

Comment:
let's name it the same as in the CPP code? `ORPHAN_TX_EXPIRE_TIME`

Reply:
fixed